cpu/gd32v/include/vendor/gd32vf103_periph.h

```diff
@@ -11549,8 +11549,8 @@ typedef struct {                                /*!< (@ 0x40002C00) WWDGT Struct
 //#define GPIOC_BASE                  0x40011000UL
 //#define GPIOD_BASE                  0x40011400UL
 //#define GPIOE_BASE                  0x40011800UL
-//#define I2C0_BASE                   0x40005400UL
-//#define I2C1_BASE                   0x40005800UL
+#define I2C0_BASE                   0x40005400UL
+#define I2C1_BASE                   0x40005800UL
```

Comment:
why were they commented out in the first place :exploding_head:

Reply:
I don't know, we have to ask @bergzand. The file seems to be generated with SDVconv from the gs32vf103.svd file and these components were probably not selected?